plugins/channelrx/demoddatv/leansdr/dvbs2.h

```diff
@@ -733,7 +736,7 @@ struct s2_frame_receiver : runnable
         // Read PLSCODE
 
         uint64_t plscode = 0;
-        complex<float> pls_symbols[plscodes.LENGTH];
+        complex<float> pls_symbols[PLSCODES_LENGTH];
```

Comment:
Alternatively you could code this that I think would work (at least it does with clang while g++ lets it go...):
```c++
complex<float> pls_symbols[plscodes<T>::LENGTH]
```
... and avoids having to define a `PLSCODES_LENGTH`. A similar construct exists at line 224